themes/default/layouts/product/self-hosted.html

```diff
@@ -55,15 +55,15 @@ <h5 class="my-6">Docker</h5>
                         <img class="h-12" src="/logos/tech/gcp-logo.svg" alt="Google Cloud" />
                     </div>
                     <h5 class="my-6">Google Cloud</h5>
-                    <a href="{{ relref $pageContext "/contact" }}" class="btn-secondary">Contact Us</a>
+                    <a href="{{ relref $pageContext "/docs/guides/self-hosted/gke-hosted" }}" class="btn-secondary">Google Docs</a>
```

Comment:
this does not appear that we are sending them to google docs, am i missing something?

Reply:
ohh lol, i think we might want to make his link text more descriptive
maybe... Google Cloud Self-hosted docs?

Reply:
I was following the naming used for the other buttons. 
So, I would want to change all the other buttons as well.
That said, I think anything more than 2 words may make for an unruly, large button.

Reply:
its just that "Google Docs" is a product...